deploy/monai-deploy-express/configs/informatics-gateway.json

```diff
@@ -43,15 +70,14 @@
       }
     },
     "storage": {
-      "bufferRootPath": "/payloads",
-      "tempStorageRootPath": "/incoming",
+      "localTemporaryStoragePath": "/payloads",
+      "remoteTemporaryStoragePath": "/incoming",
       "bucketName": "monaideploy",
       "storageRootPath": "/payloads",
       "temporaryBucketName": "monaideploy",
       "serviceAssemblyName": "Monai.Deploy.Storage.MinIO.MinIoStorageService, Monai.Deploy.Storage.MinIO",
       "watermarkPercent": 75,
       "reserveSpaceGB": 5,
-      "concurrentUploads": 4,
```

Comment:
Please do keep `concurrentUploads`, this overrides the default value of 2 or 3 I think.

Reply:
Added concurrentUploads in as suggested.